src/main/java/teammates/storage/api/FeedbackResponseCommentsDb.java

```diff
@@ -535,40 +539,42 @@ private FeedbackResponseComment getFeedbackResponseCommentEntity(
         return feedbackResponseCommentList.get(0);
     }
     
-    private List<FeedbackResponseComment> getFeedbackResponseCommentEntitiesForGiverInCourse(String courseId, String giverEmail) {
+    private List<FeedbackResponseComment> getFeedbackResponseCommentEntitiesForGiverInCourse(String courseId,
+                                                                                             String giverEmail) {
         Query q = getPM().newQuery(FeedbackResponseComment.class);
         q.declareParameters("String courseIdParam, String giverEmailParam");
         q.setFilter("courseId == courseIdParam && giverEmail == giverEmailParam");
         
         @SuppressWarnings("unchecked")
-        List<FeedbackResponseComment> feedbackResponseComments = (List<FeedbackResponseComment>) q.execute(courseId, giverEmail);
+        List<FeedbackResponseComment> feedbackResponseComments =
+                (List<FeedbackResponseComment>) q.execute(courseId, giverEmail);
         
         return feedbackResponseComments;
     }
     
-    private List<FeedbackResponseComment> getFeedbackResponseCommentEntitiesForResponse(String feedbackResponseId){
+    private List<FeedbackResponseComment> getFeedbackResponseCommentEntitiesForResponse(String feedbackResponseId) {
         Query q = getPM().newQuery(FeedbackResponseComment.class);
         q.declareParameters("String feedbackResponseIdParam");
         q.setFilter("feedbackResponseId == feedbackResponseIdParam");
         
         @SuppressWarnings("unchecked")
         List<FeedbackResponseComment> feedbackResponseCommentList =
-            (List<FeedbackResponseComment>) q.execute(feedbackResponseId);
+                (List<FeedbackResponseComment>) q.execute(feedbackResponseId);
         
         return getCommentsWithoutDeletedEntity(feedbackResponseCommentList);
     }
     
-    private List<FeedbackResponseComment> getFeedbackResponseCommentEntitiesForSession(
-            String courseId, String feedbackSessionName) {
+    private List<FeedbackResponseComment> getFeedbackResponseCommentEntitiesForSession(String courseId,
+                                                                                       String feedbackSessionName) {
         
         Query q = getPM().newQuery(FeedbackResponseComment.class);
         q.declareParameters("String courseIdParam, String feedbackSessionNameParam");
-        q.setFilter("courseId == courseIdParam && " +
-                "feedbackSessionName == feedbackSessionNameParam");
+        q.setFilter("courseId == courseIdParam && "
+                + "feedbackSessionName == feedbackSessionNameParam");
```

Comment:
This is fine but even better if you indent it further until it is on the right of `(`